src/client/pydaos/raw/conversion.py

```diff
@@ -8,6 +8,9 @@
 import ctypes
 import uuid
 
+from .. import pydaos_shim
+from .. import DAOS_MAGIC
```

Comment:
We've added addition checks for python importing since this this PR was created, can you rebase to pick up the new tests please.  I expect you'll need to update these lines as below.
```suggestion
from .. import (DAOS_MAGIC, pydaos_shim)
```

Reply:
- [ ] Update python import

Reply:
Fixed with commit 2a8c6cddf